packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -378,6 +418,9 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
                 countered: false
             })
         );
+
+        // Set the L1 head hash at the time of the game's creation.
+        l1Head = Hash.wrap(blockhash(block.number - 1));
```

Comment:
Note: This will need to change, but can be done in another PR.

Reply:
Good candidate for stacking 😉